src/schema/v2/artwork/index.ts

```diff
@@ -919,6 +919,11 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         description:
           "The string that describes domestic and international shipping.",
         resolve: (artwork) => {
+          const fullCountryName =
+            COUNTRIES[
+              artwork.shipping_origin[artwork.shipping_origin.length - 1]
```

Comment:
info: This seems like a concise but potentially brittle way of getting the country name from that map. Can we expect artwork to always have a `shipping_origin` key on it? Can we expect that `artwork.shipping_origin[artwork.shipping_origin.length - 1]` will always resolve to a key in `COUNTRIES`?